pkg/proxy/iptables/proxier.go

```diff
@@ -1155,39 +1219,14 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 
 			if hasEndpoints {
-				args = append(args[:0],
+				proxier.natRules.Write(
+					"-A", string(kubeServicesChain),
 					"-m", "comment", "--comment", fmt.Sprintf(`"%s external IP"`, svcNameString),
 					"-m", protocol, "-p", protocol,
 					"-d", externalIP,
 					"--dport", strconv.Itoa(svcInfo.Port()),
+					"-j", string(extTargetChain),
 				)
-
-				destChain := svcXlbChain
-				// We have to SNAT packets to external IPs if externalTrafficPolicy is cluster
```

Comment:
Having these comments very close to the code was really helpful to me.  Any way we can retain this level of explanation?